### PR TITLE
UG mass reformatting: have addressed some issues

### PR DESCRIPTION
**Pull request by somethingfishyfishy**
Have not marked which one is solved by this yet

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -50,77 +49,85 @@ data_coNdUctorS is a **desktop address book application for managing contact det
...
 ### Adding a contact: `add`
+Format:<br>`add n/NAME th/TELEGRAM_HANDLE e/EMAIL s/STUDENT_STATUS r/ROLE…​ [nn/NICKNAME]`
 
-Adds a contact to the address book. (labels do not need to be in order, NICKNAME is optional)
-//& LABELS OR PARAMETERS?
+<box type="definition" seamless>
 
-Format:
-* `add n/NAME th/TELEGRAM_HANDLE e/EMAIL s/STUDENT_STATUS r/ROLE…​ [nn/NICKNAME]`
+**Action:** Adds a contact to the address book. (parameters do not need to be in order, NICKNAME is optional)
```
Comment (LowXiSi) [inline docs/UserGuide.md:117]:
```suggestion
**Action:** Adds a contact to the address book. (parameters do not need to be in order, nn/NICKNAME is optional)
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -50,77 +49,85 @@ data_coNdUctorS is a **desktop address book application for managing contact det
...
+</box>
 
 <box type="tip" seamless>
 
-**Tip:** A contact can have one to many roles which match one of the following
+**Tip:** A contact must have at least one role which matches\* the following:
```
Comment (LowXiSi) [inline docs/UserGuide.md:122]: This suggests that only one of the roles need to match the role list

File: docs/UserGuide.md [hunk 3]
```diff
@@ -129,44 +136,66 @@ Examples:
 * `add n/Alex Yeoh th/alexyeoh123 e/alexyeoh@example.com ss/masters r/President r/Admin`
 ![result for 'add'](images/addAlexYeohResult.png)
 
+Note:
+* `add` command will fail if you try to add a person which already exists. You may refer to [invalid contacts](#what-is-considered-as-invalid-contacts) to see what counts as same person.
```
Comment (LowXiSi) [inline docs/UserGuide.md:140]:
```suggestion
* `add` command will fail if you try to add a person who is already in the address book. You may refer to [invalid contacts](#what-is-considered-as-invalid-contacts) to see what is considered as having the same identity.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+* `add` command will fail if you attempt to add person with a duplicate field as an existing contact, please refer to [invalid contacts](#what-is-considered-as-invalid-contacts) for more information.
```
Comment (LowXiSi) [inline docs/UserGuide.md:141]:
```suggestion
* `add` command will fail if you attempt to add a person with a duplicate field as an existing contact, please refer to [invalid contacts](#what-is-considered-as-invalid-contacts) for more information.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+<div style="page-break-after: always;"></div>
 
 ### Editing a contact: `edit`
 
-Edits an existing contact in the address book.
-In both methods below:
+<box type="definition" seamless>
+
+**Action:** Edits an existing contact in the address book. <br> You may edit by specifying the index of 
+the contact of the displayed address book that you wish to edit, or you may edit by specifying the full name
```
Comment (LowXiSi) [inline docs/UserGuide.md:150]:
```suggestion
the contact in the displayed address book, or by specifying the full name
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+of the contact that is currently displayed that you wish to edit. <br>
```
Comment (LowXiSi) [inline docs/UserGuide.md:151]:
```suggestion
of any currently displayed contact that you wish to edit. <br>
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+</box>
+
+In both methods of editing:
 * At least one of the optional fields must be provided.
```
Comment (LowXiSi) [inline docs/UserGuide.md:155]:
```suggestion
* Only contacts shown in the displayed list can be edited.
* At least one of the optional fields must be provided.
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -129,44 +136,66 @@ Examples:
...
-*  `edit 2 r/Admin r/President` Edits the roles of the 2nd displayed contact to be Admin and President, this removes all existing roles user has.
+Examples:<br>
 
-//& ACTUALLY IS IT CONSIDERED INCONSISTENT IF WE SOMETIMES WRITE NAME, THEN ALSO FULL_NAME? Might want to say FULL_NAME as indicated in the NAME field? 
-#### Edit by name
-* Edits the contact specified by `FULL_NAME`. 
-* The `FULL_NAME` has to be an exact match to an existing contact.
-* The `FULL_NAME` is non-case-sensitive.
+*  `edit 1 th/johndoe123 e/johndoe@example.com`<br>Edits the telegram handle and email address of the 1st displayed contact to be `johndoe123` and `johndoe@example.com` respectively.<br>
+*  `edit 2 r/Admin r/President` <br> Edits the roles of the 2nd displayed contact to be `Admin` and `President`, this removes all existing roles user has.
```
Comment (LowXiSi) [inline docs/UserGuide.md:169]:
```suggestion
*  `edit 2 r/Admin r/President` <br> Edits the roles of the 2nd displayed contact to be `Admin` and `President`, this replaces all existing roles of the contact.
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+*  `edit alex yeoh th/johndoe123 e/johndoe@example.com` <br> Edits the telegram handle and email address of the contact `Alex Yeoh` to be `johndoe123` and `johndoe@example.com` respectively.
 ![result for 'edit FULL_NAME'](images/editAlexYeohResult.png)
 
+<box type="tip" seamless>
+
 Note:
-* `FULL_NAME` refers to the full name of contact in the address book (to edit) while `n/NAME` is an input to change the name to (to change to).
-* `edit` command will fail if you enter a duplicate field as seen below under [invalid contacts](#what-is-considered-as-invalid-contacts)
+* `NAME` refers to the full name of contact displayed (to edit) while `n/NAME2` is an input to change the contact's name to (to change to).
+* `edit` command will fail if you enter a duplicate field that an existing contact has as seen in section [invalid contacts](#what-is-considered-as-invalid-contacts)
```
Comment (LowXiSi) [inline docs/UserGuide.md:185]:
```suggestion
* `edit` command will fail if you enter a duplicate field as an existing contact as seen in section [invalid contacts](#what-is-considered-as-invalid-contacts)
```

File: docs/UserGuide.md [hunk 10]
```diff
@@ -129,44 +136,66 @@ Examples:
...
 
+<div style="page-break-after: always;"></div>
 
 ### Deleting a contact: `delete`
+<box type="definition" seamless>
 
-Deletes the specified contact from the address book.
+**Action:**
+Deletes the specified contact from the address book.<br> You may delete by specifying the index of
+the contact of the displayed address book that you wish to delete, or you may delete by specifying the full name
```
Comment (LowXiSi) [inline docs/UserGuide.md:196]:
```suggestion
the contact in the displayed address book, or you by specifying the full name
```

File: docs/UserGuide.md [hunk 11]
```diff
@@ -129,44 +136,66 @@ Examples:
...
+of the contact that is currently displayed that you wish to delete. <br>
```
Comment (LowXiSi) [inline docs/UserGuide.md:197]:
```suggestion
of any currently displayed contact that you wish to delete. <br>
```

File: docs/UserGuide.md [hunk 12]
```diff
@@ -184,13 +213,19 @@ Examples:
 Format `delete NAME` or `delete n/NAME`
 
 * Delete the contact whose name matches the specified `NAME`.
-* The `NAME` has to be an exact match.
-* The `NAME` is non-case-sensitive.
+* The `NAME` has to be an exact match to an existing contact which is displayed.
```
Comment (LowXiSi) [inline docs/UserGuide.md:216]:
```suggestion
* The `NAME` has to be an exact match to an existing contact in the displayed list.
```

File: docs/UserGuide.md [hunk 13]
```diff
@@ -200,28 +235,35 @@ Examples:
...
 //& ISSUE 261
 
-Finds contacts whose details matches all given fields.
-
 Format: `find [n/NAME] [th/TELEGRAM_HANDLE] [e/EMAIL] [ss/STUDENT_STATUS] [r/ROLE]…​ [nn/NICKNAME]`
+<box type="definition" seamless>
+
+**Action:** Finds contacts whose details matches all given fields.
+* Each PREFIX can only be used once (except Role)
+* All parameters are not case-sensitive. e.g. `find r/pResiDent` &rarr; valid
```
Comment (LowXiSi) [inline docs/UserGuide.md:249]:
```suggestion
* All fields are not case-sensitive. e.g. `find r/pResiDent` &rarr; valid
```

File: docs/UserGuide.md [hunk 14]
```diff
@@ -294,13 +356,15 @@ that they have at least one click on the pagination before using keyboard to mak
...
+<div style="page-break-after: always;"></div>
+
 ### Showing total number of contacts -- Footer status bar
 
 A status bar that contains information about the number of contacts in the list and the address book data file path is attached as a footer
 at the bottom of the app. It shows the total number of contacts being listed in the app.
 
 Examples:
-* `find n/Adam` may show only 1 contact out of 19.
+* `find n/Adam` may show only 1 contact out of 19. 1 refers to the found contact, 19 refers to the total contacts in address book.
```
Comment (LowXiSi) [inline docs/UserGuide.md:367]:
```suggestion
* `find n/Adam` may show only 1 contact out of 19. 1 refers to the number of found contacts, 19 refers to the total number of contacts in address book.
```

**Review by LowXiSi — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (WinstonJin) [reply docs/UserGuide.md:122]: **Tip:** A contact must have at least one role. Roles should match\* the following:

File: docs/UserGuide.md [hunk 15]
```diff
@@ -50,77 +49,85 @@ data_coNdUctorS is a **desktop address book application for managing contact det
...
 
-Format:
-* `add n/NAME th/TELEGRAM_HANDLE e/EMAIL s/STUDENT_STATUS r/ROLE…​ [nn/NICKNAME]`
+**Action:** Adds a contact to the address book. (parameters do not need to be in order, nn/NICKNAME is optional)
+</box>
 
 <box type="tip" seamless>
 
-**Tip:** A contact can have one to many roles which match one of the following
+**Tip:** A contact must have at least one role which matches\* the following:
```
Comment (WinstonJin) [inline docs/UserGuide.md:122]:
```suggestion
**Tip:** A contact must have at least one role. Roles should match\* the following:
```

**Review by WinstonJin — APPROVED** [review]
Approved all of Xi Si's changes, good to merge

**Pull request merged**